Specs/DateTime/Japanese/DateExtractor.json

```diff
@@ -1,3020 +1,3254 @@
-        "Type": "date",
-        "Start": 0,
-        "Length": 10
-      }
-    ]
-  },
-  {
-    "Input": "今月の十日",
-    "NotSupported": "dotnet, javascript, python, java",
-    "Results": [
-      {
-        "Text": "今月の十日",
-        "Type": "date",
-        "Start": 0,
-        "Length": 5
-      }
-    ]
-  },
-  {
-    "Input": "明後日、空いていますか？",
-    "NotSupported": "dotnet, javascript, python, java",
-    "Results": [
-      {
-        "Text": "明後日、空いていますか？",
-        "Type": "date",
-        "Start": 0,
-        "Length": 12
-      }
-    ]
-  },
-  {
-    "Input": "コルタナさん、2018-7-6 金曜日にジムとの skype を設定して。",
-    "Context": {
-      "ReferenceDateTime": "2018-07-06T00:00:00"
-    },
-    "NotSupported": "dotnet, javascript, python, java",
-    "Results": [
-      {
-        "Text": "2018-7-6 金曜日",
-        "Type": "date",
-        "Start": 7,
-        "Length": 12
-      }
-    ]
-  },
-  {
-    "Input": "コルタナさん、7.6 金曜日にジムとの skype を設定して。",
-    "Context": {
-      "ReferenceDateTime": "2018-07-06T00:00:00"
-    },
-    "NotSupported": "dotnet, javascript, python, java",
-    "Results": [
-      {
-        "Text": "7.6 金曜日",
-        "Type": "date",
-        "Start": 7,
-        "Length": 7
-      }
-    ]
-  },
-  {
-    "Input": "367年1月1日",
-    "NotSupported": "dotnet, javascript, python, java",
-    "Results": [
-      {
-        "Text": "367年1月1日",
-        "Type": "date",
-        "Start": 0,
-        "Length": 8
-      }
-    ]
-  },
-  {
-    "Input": "もうすぐ1月19日になります。",
-    "NotSupported": "dotnet, javascript, python, java",
-    "Results": [
-      {
-        "Text": "1月19日",
-        "Type": "date",
-        "Start": 4,
-        "Length": 5
-      }
-    ]
-  },
-  {
-    "Input": "1月19日",
-    "NotSupported": "dotnet, javascript, python, java",
-    "Results": [
-      {
-        "Text": "1月19日",
-        "Type": "date",
-        "Start": 0,
-        "Length": 5
-      }
-    ]
-  },
-  {
-    "Input": "コルタナさん、7/6 金曜日にジムとの skype を設定して。",
-    "Context": {
-      "ReferenceDateTime": "2018-07-06T00:00:00"
-    },
-    "NotSupported": "dotnet, javascript, python, java",
-    "Results": [
-      {
-        "Text": "7/6 金曜日",
-        "Type": "date",
-        "Start": 7,
-        "Length": 7
-      }
-    ]
-  },
-  {
-    "Input": "最近はお元気ですか？",
-    "NotSupported": "dotnet, javascript, python, java",
-    "Results": [
-      {
-        "Text": "最近",
-        "Type": "date",
-        "Start": 0,
-        "Length": 2
-      }
-    ]
-  },
-  {
-    "Input": "旧暦の2015年お正月の1日は春節です。",
-    "NotSupported": "dotnet, javascript, python, java",
-    "Results": [
-      {
-        "Text": "旧暦の2015年お正月の1日",
-        "Type": "date",
-        "Start": 0,
-        "Length": 14
-      }
-    ]
-  },
-  {
-    "Input": "一昨昨日はどうでしたか？",
-    "NotSupported": "dotnet, javascript, python, java",
-    "Results": [
-      {
-        "Text": "一昨昨日",
-        "Type": "date",
-        "Start": 0,
-        "Length": 4
-      }
-    ]
-  }
+[

```

Comment:
Is this a line break encoding issue? Please be careful to not convert line breaks unless necessary, otherwise it makes reviewing changes much harder.
Thanks.